SimpleCam/SimpleCam/SimpleCam.m

```diff
@@ -303,6 +313,17 @@ - (void) setup {
     
     // -- PREPARE OUR CONTROLS -- //
     [self loadControls];
+    
+    // -- PREPARE CAMERA FLASH ANIMATION -- //
+    if (cameraCaptureFlashAnimation) {
```

Comment:
I don't think this needs to be in the if block, should be able to safely call these methods.